examples/azure-tf-example/README.md

```diff
@@ -0,0 +1,253 @@
+```bash
+$ porter credentials generate
+Generating new credential porter-terraform-example from bundle porter-terraform-example
+==> 4 credentials required for bundle porter-terraform-example
+? How would you like to set credential "client_id"  [Use arrows to move, space to select, type to filter]
+  specific value
+> environment variable
+  file path
+  shell command
+```
+
+For each of the credentials, provide the corresponding environment variable that you set above. For example, for `client_id`, select `environment variable` and provide the value `AZURE_CLIENT_ID`.
+
+```bash
+$ porter credentials generate
+Generating new credential porter-terraform-example from bundle porter-terraform-example
+==> 4 credentials required for bundle porter-terraform-example
+? How would you like to set credential "client_id" environment variable
+? Enter the environment variable that will be used to set credential "client_id" AZURE_CLIENT_ID
+? How would you like to set credential "client_secret" environment variable
+? Enter the environment variable that will be used to set credential "client_secret" AZURE_CLIENT_SECRET
+? How would you like to set credential "subscription_id" environment variable
+? Enter the environment variable that will be used to set credential "subscription_id" AZURE_SUBSCRIPTION_ID
+? How would you like to set credential "tenant_id" environment variable
+? Enter the environment variable that will be used to set credential "tenant_id" AZURE_TENANT_ID
+Saving credential to /Users/jeremyrickard/.porter/credentials/porter-terraform-example.yaml
+```
+
+## Installing the Bundle
+
+Once you have built the bundle and generated a credential, you're ready to install the bundle! To do that, you'll use the `porter install` command:
+
+```bash
+$ porter install -c porter-terraform-example
+installing porter-terraform-example...
+executing porter install configuration from /cnab/app/porter.yaml
+Create an Azure Storage Account
+Starting deployment operations...
+Finished deployment operations...
+Emit the key in base64 encoded form
+Here is a the storage account key (base64 encoded) ==> %%A KEY%%
+Create Azure CosmosDB and Event Hubs
+Initializing Terraform...
+/usr/bin/terraform terraform init -backend=true -backend-config=access_key=******* -backend-config=container_name=portertf -backend-config=key=porter-terraform-example.tfstate -backend-config=storage_account_name=porterstorage -reconfigure
+
+Initializing the backend...
+
+Successfully configured the backend "azurerm"! Terraform will automatically
+use this backend unless the backend configuration changes.
+
+Initializing provider plugins...
+- Checking for available provider plugins on https://releases.hashicorp.com...
+- Downloading plugin for provider "azurerm" (1.31.0)...
+
+< OUTPUT TRUNCATED>
+```
+
+Installing the bundle will take some amount of time. As it is running you should see an output line like this:
+
+```
+Here is a the storage account key (base64 encoded) ==> <SOME STRING>
```

Comment:
Can we add this as an output to the bundle/porter manifest, so that the user can grab the output after install via the appropriate porter cli command?  Or does the azure mixin not yet support outputs?

Reply:
Yeah, should be able to update that. The bundle outputs weren't ready when i did this :)

Reply:
Actually, @vdice since we havent released Porter with bundle outputs, i'd prefer to rev the example later?

Reply:
Sgtm!